catkit/testbed/caching.py

```diff
@@ -236,6 +237,22 @@ def __getattr__(self, item):
         device = devices[member]
         return device.__getattribute__(item)
 
+    def __setattr__(self, name, value):
+        if "_DeviceCacheEnum__object_exists" in self.__dict__:
+            config_id = object.__getattribute__(self, "config_id")
```

Comment:
Copy and paste error, this is unnecessary and should just be ``config_id = self. config_id``.